*** GCDAsyncUdpSocket.m	2016-03-08 08:47:26.000000000 -0800
--- GCDAsyncUdpSocket2.m	2016-03-07 14:18:33.000000000 -0800
***************
*** 1950,1955 ****
--- 1950,1966 ----
  		int status;
  		
  		// Set socket options
+ 
+         int reuseport = 1;
+   		status = setsockopt(socketFD, SOL_SOCKET, SO_REUSEPORT, &reuseport, sizeof(reuseport));
+   		if (status == -1)
+   		{
+   			if (errPtr)
+   				*errPtr = [self errnoErrorWithReason:@"Error enabling port reuse (setsockopt)"];
+ 
+   			close(socketFD);
+   			return SOCKET_NULL;
+   		}
  		
  		status = fcntl(socketFD, F_SETFL, O_NONBLOCK);
  		if (status == -1)
